enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/config/EnvVarConfig.java

```diff
@@ -119,6 +124,10 @@ public String getTrustedAdapterCertsPath() {
         return trustedAdapterCertsPath;
     }
 
+    public boolean getTrustDefaultCerts() {
```

Comment:
Shall we rename this method as `isTrustDefaultCerts`. Because the name 'getTrustDefaultCerts' is little bit confusing as it is retrieving a boolean config value.

Reply:
+1